extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -517,6 +541,49 @@ export class AmpConsent extends AMP.BaseElement {
     });
   }
 
+  /**
+   * Sync consent string and consent state value to local storage.
+   */
+  syncConsentStringAndStateValue_() {
+    const responsePromise = this.getConsentRemote_();
+    responsePromise.then(response => {
+      if (
+        response &&
+        typeof this.consentConfig_['consentRequired'] === 'boolean' &&
+        !this.consentStateChangedViaPromptUI_ &&
+        isExperimentOn(this.win, 'amp-consent-geo-override')
+      ) {
+        this.consentStateManager_
+          .getLastConsentInstanceInfo()
+          .then(storedInfo => {
+            this.updateCacheIfNotNull_(storedInfo, response);
+          });
+      }
+    });
+  }
+
+  /**
+   * Sync cache is the value is not null. Defaults to storedValue.
+   * @param {./consent-info.ConsentInfoDef} storedInfo
+   * @param {?JsonObject} response
+   */
+  updateCacheIfNotNull_(storedInfo, response) {
+    const stateValue =
+      response['consentStateValue'] !== null &&
+      response['consentStateValue'] !== undefined
+        ? convertValueToState(response['consentStateValue'])
+        : storedInfo['consentState'];
```

Comment:
because `storedInfo` is a type of  `ConsentInfoDef ` you should access this field via
`storeInfo.consentState` in case complier minifies member fields.